src/code.cloudfoundry.org/cf-tcp-router/config/config_test.go

```diff
@@ -296,51 +304,47 @@ var _ = Describe("Config", Serial, func() {
 				}))
 			})
 
-			It("writes the correct cert and key files with correct permissions", func() {
-				for i, name := range []string{"prod", "dev"} {
+			It("does not write the correct cert and key files as enableCertCreation is false", func() {
```

Comment:
I am confused about the wording of the test above (but I can't comment on it because it didn't change in this PR). 

The test on line 292 says:
`It("adds the certs and keys to the expected directories", func() {`

Then this test says `does not write the correct cert and key files as enableCertCreation is false"`
Those seem contradictory to me. 

Please clarify.